cinema/views.py

```diff
@@ -48,6 +89,40 @@ class MovieSessionViewSet(viewsets.ModelViewSet):
     queryset = MovieSession.objects.all()
     serializer_class = MovieSessionSerializer
 
+    @staticmethod
+    def _params_to_ints(qs):
+        return [int(str_id) for str_id in qs.split(",")]
```

Comment:
make ur code DRY u have already implement this method, if u need to use it in few classes u can create mixin for this

Reply:
done